### PR TITLE
[Dashboard] envs clean-up

### PR DESCRIPTION
**Pull request by kien-ngo**
- [x] Remove a lof of duplicated instances of `ThirdwebClient` & env variables across multiple files
- [x] There's only 1 main ThirdwebClient from now on which can be imported from `lib/thirdweb-client`. It can switch between clientId and secretKey depends on the environment
- [x] All env variables like `DASHBOARD_THIRDWEB_CLIENT_ID`, `DASHBOARD_THIRDWEB_SECRET_KEY`, `isProd` etc. now live inside `@/constants/env`

---

## PR-Codex overview
This PR updates import paths and removes redundant imports. 

### Detailed summary
- Updated import paths for `thirdwebClient` in multiple files
- Removed redundant imports of `isProd` from `constants/rpc`
- Updated usage of `isProd` from `constants/rpc` to `@/constants/env`

> The following files were skipped due to too many changes: `apps/dashboard/src/contract-ui/tabs/overview/page.tsx`, `apps/dashboard/src/contract-ui/tabs/nfts/components/airdrop-tab.tsx`, `apps/dashboard/src/components/app-layouts/app.tsx`, `apps/dashboard/src/components/app-layouts/provider-setup.tsx`, `apps/dashboard/src/contract-ui/tabs/manage/page.tsx`, `apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/opengraph-image.tsx`, `apps/dashboard/src/lib/sdk.ts`, `apps/dashboard/src/@/constants/client.ts`, `apps/dashboard/src/app/(dashboard)/(chain)/components/server/chain-icon.tsx`, `apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/FaucetButton.tsx`, `apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/tx/[txHash]/page.tsx`, `apps/dashboard/src/pages/api/moralis/balances.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by joaquim-verges — APPROVED** [review]

File: apps/dashboard/src/lib/thirdweb-client.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
-import { DASHBOARD_THIRDWEB_SECRET_KEY } from "@/constants/env";
-import { DASHBOARD_THIRDWEB_CLIENT_ID } from "constants/rpc";
+import {
```
Comment (jnsdls) [inline apps/dashboard/src/lib/thirdweb-client.ts:1]: removing the duplication here makes sense but can we move it the *opposite* direction? I'd like to keep `@constants/client` and remove this file instead.

Comment (jnsdls) [reply apps/dashboard/src/lib/thirdweb-client.ts:1]: maybe even `@/client`

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/contract-ui/tabs/overview/components/MarketplaceDetails.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ import {
 } from "tw-components";
 import { AddressCopyButton } from "tw-components/AddressCopyButton";
 import { NFTMediaWithEmptyState } from "tw-components/nft-media";
-import { thirdwebClient } from "../../../../lib/thirdweb-client";
+import { thirdwebClient } from "../../../../@/constants/client";
```
Comment (jnsdls) [inline apps/dashboard/src/contract-ui/tabs/overview/components/MarketplaceDetails.tsx:37]: fix import url


```suggestion
import { thirdwebClient } from "@/constants/client";
```

File: apps/dashboard/src/lib/ens.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { isAddress } from "thirdweb";
 import { resolveAddress, resolveName } from "thirdweb/extensions/ens";
 import invariant from "tiny-invariant";
-import { thirdwebClient } from "./thirdweb-client";
+import { thirdwebClient } from "../@/constants/client";
```
Comment (jnsdls) [inline apps/dashboard/src/lib/ens.ts:4]: absolute path pls


```suggestion
import { thirdwebClient } from "@/constants/client";
```

**Review by jnsdls — CHANGES_REQUESTED** [review]

**Review by jnsdls — APPROVED** [review]

**Pull request merged**